src/components/Cards/EventCard/EventCard.tsx

```diff
@@ -183,78 +233,139 @@ const EventCard: React.FC<EventCardProps> = (props) => {
-            <Link href={"#/eventDetails/" + props.id} className="link-nothing">
-              <Card className="mt-2 mb-2 shadow card card-background">
-                <Card.Img
-                  src={props.image}
-                  alt="Card Image"
-                  className="card_image"
-                  referrerPolicy="no-referrer"
+        // <Link href={"#/eventDetails/" + props.id} className="link-nothing">
+        <Card className="" style={{ boxShadow: "none" }}>
+          <CardMedia
+            component="img"
+            image={props.image}
+            alt="Card Image"
+            className="card_image"
+            style={{ height: "224px" }}
+          />
+
+          <CardContent
+            style={{
+              padding: "16px",
+            }}
+          >
+            <Typography
+              style={{
+                fontSize: theme.fontSize.subheading,
+                fontWeight: 800,
+              }}
+              component="div"
+            >
+              {props.title}
+            </Typography>
+            <div
+              className="link-primary"
+              style={{
+                backgroundColor: theme.colors.primary,
+                color: theme.colors.lightBackground,
+              }}
+            >
+              <a href={"#/profile/" + props.host}>{props.hostName}</a>
+            </div>
+            <p className="card-description"> {props.description}</p>
+            <div className="d-flex align-items-center ">
+              <EventAvailableIcon
+                style={{ fontSize: "26px", color: theme.colors.primary }}
+              />
+              <span className="card-time">
+                <span className="ms-2">{props.date}</span>
+                <span className="ms-2">
+                  {props.time.replace(/\s\(Indian Standard Time\)/, "")}
+                </span>
+              </span>
+              {!props.isDashboard ? (
+                new Date(props.date) > new Date() ? (
+                  <Button className="btn-c  m-3" onClick={registerForEventX}>
+                    Register
+                  </Button>
+                ) : (
+                  <Button
+                    style={{
+                      padding: "8px 16px",
+                      borderRadius: "8px",
+                      color: "black",
+                      textTransform: "none",
+                      fontWeight: 700,
+                      fontSize: "16px",
+                    }}
+                    className="btn-d  m-2"
+                  >
+                    Expired
+                  </Button>
+                )
+              ) : null}
+            </div>
+            <hr
+              style={{
+                color: theme.colors.secondaryLight,
+                border: "1px solid",
+              }}
+            />
+            <Stack direction="row" spacing={1}>
+              {props.tags.split(",").map((tag, index) => (
+                <Chip
```

Comment:
Make this clickable. [https://mui.com/material-ui/react-chip/#clickable](https://mui.com/material-ui/react-chip/#clickable)